ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/version/withoutledgerid/VersionClient.scala

```diff
@@ -3,14 +3,26 @@
 
 package com.daml.ledger.client.services.version.withoutledgerid
 
-import com.daml.ledger.api.domain.LedgerId
-import com.daml.ledger.api.v1.version_service.GetLedgerApiVersionRequest
+import com.daml.ledger.api.domain.{Feature, LedgerId}
+import com.daml.ledger.api.v1.experimental_features.CommandDeduplicationPeriodSupport.{
+  DurationSupport,
+  OffsetSupport,
+}
+import com.daml.ledger.api.v1.experimental_features.{
+  CommandDeduplicationFeatures,
+  CommandDeduplicationPeriodSupport,
+  CommandDeduplicationType,
+  ExperimentalFeatures,
+  ExperimentalStaticTime,
+}
+import com.daml.ledger.api.v1.version_service.{FeaturesDescriptor, GetLedgerApiVersionRequest}
 import com.daml.ledger.api.v1.version_service.VersionServiceGrpc.VersionServiceStub
 import com.daml.ledger.client.LedgerClient
 import scalaz.syntax.tag._
 
 import scala.concurrent.{ExecutionContext, Future}
 
+@SuppressWarnings(Array("org.wartremover.warts.Option2Iterable"))
```

Comment:
I don't know how to avoid that warning, happy to take suggestions. It comes from calling `toSeq` on an `Option`, I'm pretty sure. I guess when this conversion is inserted implicitly it's a code smell, but I don't think wartremover is precise enough to catch the difference (between this being explicitly called or implicitly by the compiler).

Reply:
The warning tells you how to fix it:
> ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/version/withoutledgerid/VersionClient.scala:68: error: [wartremover:Option2Iterable] Implicit conversion from Option to Iterable is disabled - use Option#toList instead
Switching `toSeq` to `toList` makes the file compile.

Reply:
Fine, that works, but the error is still wrong. I'm writing an explicit toSeq, not an implicit one.

Reply:
From the docs of [`Option.toSeq`](https://www.scala-lang.org/api/2.13.6/scala/Option.html#toSeq:Seq[A]):

> This member is added by an implicit conversion from Option[A] toIterable[A] performed by method option2Iterable in scala.Option.

So I’m not sure it’s wrong in this case, the only reason you can call it is because of the implicit conversion. But yeah in general wartremover definitely produces false positives but if there is an easy way to avoid them we generally opt for that over disabling the warning.

Reply:
Thanks for pointing out the workaround. I had just added the exception as that seemed like the easiest way out. Will fix so that it fits in with the code style. Now go back to your weekend! We can polish this on Monday as well.